packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1935,6 +1935,10 @@ export const components = ({
 
           [`&.${menuItemClasses.root}-destructive`]: {
             color: odysseyTokens.TypographyColorDanger,
+
+            ".MuiTypography-root": {
+              color: "inherit",
+            },
```

Comment:
Do we have to target the class? And if so, can we import that className instead?

If we're using `MuiTypography` inside of `MuiMenuItem` can we pass a class to it, or is this all MUI's code and that's why we can't avoid using a `className`?

Reply:
Not sure how to import the classname but, can certainly look into it. 

We are still using `MuiMenuItem` but, in the case where we used `MuiTypography` inside the menu item, the more specific typography styling was being applied